src/disc/disc.py

```diff
@@ -116,18 +154,18 @@ async def get_role_name_by_id(
     id: str = guild_id
 ) -> str:
     roles = await get_all_roles(id)
-    return roles[rid]
+    return roles[rid][0]
```

Comment:
why are we adding a `[0]` when `get_all_roles` hasn't changed?

Reply:
This returns a string instead of the list containing the name and the permissions and aligns with the return type described.